docs/hub/repositories-licenses.md

```diff
@@ -62,6 +62,7 @@ A full list of the available licenses is available here:
 | ISC                                                            | `isc`                                    |
 | LaTeX Project Public License v1.3c                             | `lppl-1.3c`                              |
 | Microsoft Public License                                       | `ms-pl`                                  |
+| Apple Sample Code license                                      | `apple-ascl`                             | <!-- license: https://github.com/apple/ml-tic-clip/blob/main/LICENSE -->
```

Comment:
It's unfortunate that it's a PDF, but maybe `info: https://developer.apple.com/support/downloads/terms/apple-sample-code/Apple-Sample-Code-License.pdf` would ensure more longevity? (Repo licenses have changed in the past in several cases)

Reply:
pdf is fine IMO